staging/src/k8s.io/cli-runtime/pkg/resource/mapper.go

```diff
@@ -64,6 +65,10 @@ func (m *mapper) infoForData(data []byte, source string) (*Info, error) {
 		}
 		mapping, err := restMapper.RESTMapping(gvk.GroupKind(), gvk.Version)
 		if err != nil {
+			if _, ok := err.(*meta.NoKindMatchError); ok {
```

Comment:
This is a good improvement, but is “Warning” the right term to use for this?  I’m guessing returning an error will ultimately cause the command to fail with a non zero exit code.  I’m wondering if the warning part should just be removed, since it fails outright unless I’m missing something.

Either way, I think it is convention for go errors to start with lower case.

Reply:
That's a good call

/hold